### PR TITLE
Redirect requests to root, to UI

### PR DESCRIPTION
**Pull request by kakkoyun**
This PR adds additional handlers to redirect incoming requests to root, to UI.

cc @metalmatze @squat  @aditya-konarde

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (squat) [inline run.sh]: This file seems unrelated to the title of the PR. is it included by mistake?

**Comment by kakkoyun** [comment]
@squat I thin we can merge this anyway. It doesn't harm to have a redirect to UI when you hit root path.

**Comment by kakkoyun** [comment]
This PR needs to be updated after changes in https://github.com/observatorium/observatorium/pull/17  got merged.

File: internal/server/server.go [hunk 2]
```diff
@@ -39,28 +39,35 @@ func New(logger log.Logger, reg *prometheus.Registry, opts ...Option) Server {
...
 
+	ins := newInstrumentationMiddleware(reg)
+	p := prober.New(logger)
+
 	registerProber(r, p)
 
+	uiPath := "/ui/metrics/v1"
+
+	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		http.Redirect(w, r, uiPath+"/graph", http.StatusMovedPermanently)
```
Comment (squat) [inline internal/server/server.go:61]: consider `path.Join(uiPath, "graph")`
